pyrefly/lib/commands/check.rs

```diff
@@ -487,6 +487,63 @@ impl Args {
         )
     }
 
+    pub fn run_once_with_snippet(
+        self,
+        code: String,
+        _files_to_check: FilteredGlobs,
```

Comment:
Since this parameter is unused, it seems simpler to just remove it and adjust the function call in main.rs accordingly.